frontend/src/metabase/auth/components/AuthLayout/AuthLayout.styled.tsx

```diff
@@ -47,7 +47,7 @@ export const LayoutIllustration = styled.div<{
   background-image: ${({ backgroundImageSrc }) =>
     `url("${backgroundImageSrc}")`};
   background-size: ${({ isDefault }) =>
-    isDefault ? "max(2592px, 100%) auto" : "cover"};
+    isDefault ? "max(2592px, 100%) auto" : "100% auto"};
```

Comment:
It's been [decided on Slack](https://metaboat.slack.com/archives/C063Q3F1HPF/p1711568521316139?thread_ts=1710939919.435909&cid=C063Q3F1HPF)